packages/main/src/i18n/messagebundle.properties

```diff
@@ -165,6 +165,11 @@ DATERANGEPICKER_POPOVER_ACCESSIBLE_NAME=Choose Date Range
 
 DELETE=Delete
 
+#XFLD: A - to indicate that there is an empty space. In some languages there might be a different symbol used.
```

Comment:
I suggest:
En dash (–) character to indicate that there is an empty space.
or
En dash character to indicate that there is an empty space